test/integration/add_comments_to_rejected_project_test.rb

```diff
@@ -3,6 +3,8 @@
 
 class AddCommentsToRejectedProjectTest < ActionDispatch::IntegrationTest
   test 'normal user can not edit when project in approved state' do
+    skip # TODO: fix me?
```

Comment:
do we need check this requirement with BA team?

Reply:
Good shout. I think this was a temporary `skip` that never got removed...